app/components/job/details/job-properties.component.ts

```diff
@@ -16,6 +16,11 @@ export class JobPropertiesComponent implements OnDestroy {
     @Input()
     public set job(job: Job) {
         this._job = job;
+        if (job && job.executionInfo) {
+            this.hasStartTime = !!job.executionInfo.startTime;
```

Comment:
You can do `Boolean(job.executionInfo.startTime)` I find it is clearer that double negation